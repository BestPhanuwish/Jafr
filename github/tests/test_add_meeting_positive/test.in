2
Eat breakfast
01/08/23
08:00
n
6