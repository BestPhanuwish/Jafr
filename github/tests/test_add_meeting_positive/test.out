Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!

You have the following meetings today!

You have the following meetings scheduled over the next week!

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Please enter a meeting description:
Please enter a date:
Please enter a time:
Ok, I have added Eat breakfast on 01/08/23 at 08:00.
Would you like to share this meeting? [y/n]: What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
